Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/ErrorLogBean.java

```diff
@@ -121,6 +122,7 @@ public void setToDate(Date toDate) {
 
     public void search() {
         eventsList = service.search(selectedService, selectedException, fromDate, toDate);
+        msgEventResult = CollectionUtils.isNotEmpty(eventsList) ? "Event record(s) found." : "No event record found.";
```

Comment:
Do we want to say how many were found? Just saying "Event records found" isn't really meaningful.